test/acceptance-tests/operator_images_test.go

```diff
@@ -0,0 +1,79 @@
+package acceptance_tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/securesign/structural-tests/test/support"
+	"log"
+	"regexp"
+)
+
+var _ = Describe("Trusted Artifact Signer Operator", Ordered, func() {
+
+	var (
+		snapshotImages support.SnapshotMap
+		operatorImages support.OperatorMap
+		operator       string
+	)
+
+	It("get and parse snapshot.json file", func() {
+		var err error
+		snapshotImages, err = support.ParseSnapshotImages()
+		Expect(err).NotTo(HaveOccurred())
+		Expect(snapshotImages).NotTo(BeEmpty())
+	})
+
+	It("get operator image", func() {
+		operator = snapshotImages[support.OperatorImageKey]
+		Expect(operator).NotTo(BeEmpty())
+		log.Printf("Using %s\n", operator)
+	})
+
+	It("get all TAS images used by this operator", func() {
+		iCount := support.GetEnvOrDefaultInt(support.EnvOperatorImagesCount, support.ExpectedOperatorImagesCount)
+
+		helpLogs, err := support.RunImage(operator, []string{"-h"})
+		Expect(err).NotTo(HaveOccurred())
+
+		operatorImages = support.ParseOperatorImages(helpLogs)
+		support.LogMap(fmt.Sprintf("Operator TAS images (%d):", len(operatorImages)), operatorImages)
+		Expect(len(operatorImages)).To(BeNumerically("==", iCount), "Expected to have %d images", iCount)
+	})
+
+	It("operator images are all valid", func() {
```

Comment:
All images should be `redhat.io`. Add the test, please.

Reply:
Done.